src/components/SlateEditor/plugins/embed/SlateFigure.tsx

```diff
@@ -31,6 +34,33 @@ interface ChangesProp {
   [x: string]: string | undefined;
 }
 
+export const FigureInfo = styled.div`
+  margin-bottom: ${spacing.small};
+  font-family: ${fonts.sans};
+  color: ${colors.text.primary};
+  ${fonts.sizes("16px", "24px")};
```

Comment:
Kunne vi heller brukt   _${fonts.size.text.metaText.small}_?

Reply:
Sida dette brukes til en knapp, er det kanskje "riktigere" å bruke fonts.size.text.button

Reply:
Trodde ikke dette var en knapp men et sammendrag

Reply:
Det er en knapp kun for å trigge redigeringsmodal. Ser i LicenseText som brukes for framvisninga så er det spesifisert `${fonts.sizes("18px", "24px")};` så kan endre til fonts.size.metaText.medium.